From 43e1dafe9b4f855cedff07fc44ece52e52623397 Mon Sep 17 00:00:00 2001
From: Gerd Hoffmann <kraxel@redhat.com>
Date: Fri, 10 Jan 2025 13:58:08 +0100
Subject: [PATCH 21/22] OvmfPkg/MicroVM: use PeiPcdLib for PEI_CORE

Since commit d64d1e195ceb ("MdeModulePkg: PeiMain: Introduce
implementation of delayed dispatch") the pei core wants read PCDs, so
the NULL lib doesn't cut it any more.

Signed-off-by: Gerd Hoffmann <kraxel@redhat.com>
(cherry picked from commit d6f828b3849e5adcf9a04d3c16e79388db9b3614)
---
 OvmfPkg/Microvm/MicrovmX64.dsc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/OvmfPkg/Microvm/MicrovmX64.dsc b/OvmfPkg/Microvm/MicrovmX64.dsc
index a48480f0721a..d0b8d5b54b1d 100644
--- a/OvmfPkg/Microvm/MicrovmX64.dsc
+++ b/OvmfPkg/Microvm/MicrovmX64.dsc
@@ -291,6 +291,7 @@ [LibraryClasses.common.PEI_CORE]
 !else
   DebugLib|OvmfPkg/Library/PlatformDebugLibIoPort/PlatformRomDebugLibIoPort.inf
 !endif
+  PcdLib|MdePkg/Library/PeiPcdLib/PeiPcdLib.inf
   PeCoffLib|MdePkg/Library/BasePeCoffLib/BasePeCoffLib.inf
 
 [LibraryClasses.common.PEIM]
-- 
2.48.1

